apps/opik-backend/src/main/java/com/comet/opik/api/filter/ExperimentsComparisonField.java

```diff
@@ -7,10 +7,11 @@
 @Getter
 public enum ExperimentsComparisonField implements Field {
 
-    INPUT(INPUT_QUERY_PARAM, FieldType.STRING),
-    EXPECTED_OUTPUT(EXPECTED_OUTPUT_QUERY_PARAM, FieldType.STRING),
+    INPUT(NEW_INPUT_QUERY_PARAM, FieldType.STRING),
+    EXPECTED_OUTPUT(NEW_EXPECTED_OUTPUT_QUERY_PARAM, FieldType.STRING),
     OUTPUT(OUTPUT_QUERY_PARAM, FieldType.STRING),
-    METADATA(METADATA_QUERY_PARAM, FieldType.DICTIONARY),
+    METADATA(NEW_METADATA_QUERY_PARAM, FieldType.DICTIONARY),
+    CUSTOM_FIELD(NEW_CUSTOM_FIELD, FieldType.CUSTOM_FIELD),
```

Comment:
Let's call this `DATA_FIELD`. Custom isn't significative at all and it doesn't follow the existing convention.